docs/plugin_manager.md

```diff
@@ -1,18 +1,29 @@
 Plugin Manager
 ===================
 
-## Everything is a PLUGIN in Remix
+![](images/a-plugin-man-overview.png)
 
-In order to integrate new tools made by us and by ...you into Remix, we've now made everything a plugin.
-This architecture will also allow Remix or just parts of Remix to be integrated into other projects (your's for example).   
+In Remix IDE you only load the functionality you need.  Controlling which plugins are active or inactive happens in the **Plugin Manager**.
 
-This means that you only load the functionality you need.  
+This plugin architecture has made it possible to integrate tools made by the Remix team with tools made by external teams. This architecture also allows Remix or just parts of Remix to be integrated into other projects (your's for example).   
```

Comment:
`to integrate tools made by the Remix team with tools made by external teams` can be updated as `to integrate tools developed by the Remix and external teams`

Reply:
What does `your's for example` mean?